.github/workflows/publish-img.yaml

```diff
@@ -57,6 +74,17 @@ jobs:
           tags: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ github.ref_name }}
           file: Dockerfile
 
+      - name: Push stable passt binding cni container image
+        if: startsWith(github.ref, 'refs/tags/')
+        uses: docker/build-push-action@v5.3.0
+        with:
+          context: .
+          push: true
+          tags: ${{ env.REGISTRY }}/${{ env.PASST_BINDING_CNI_IMAGE_NAME }}:${{ github.ref_name }}
```

Comment:
```suggestion
          tags: ${{ env.REGISTRY }}/${{ env.PASST_BINDING_CNI_IMAGE_NAME }}:${{ env.KUBEVIRT_VERSION }}
```

Reply:
we need the same tag, because this is how CNAO will know to consume it
once we tag, we release kubevirt-ipam-controller and this new image, and this what triggers CNAO 
to auto bump it

CNAO will deploy the new daemonSet under the same kubevirt ipam controller flag

if you want we can push another image as the KUBEVIRT_VERSION, but not just that

Reply:
ahh we need to match kubevirt-ipam-controller with passt image for CNAO to work seamless, puff is not really optional but ok.